### DISCUSSION TITLE
建议在编程版块专门开辟区域专注写漏洞修复代码，比如注入漏洞修复建议代码，xss漏洞建议修复代码，参考文档我可以提供

### DISCUSSION THREAD

**Discussion by Asura88** [he3-app/he3-feedback#185] [General]
建议在编程版块专门开辟区域专注写漏洞修复代码，比如注入漏洞修复建议代码，xss漏洞建议修复代码，参考文档我可以提供

**Comment by lyzhang1999** [comment]
非常棒的建议。He3 支持自己来写工具，如果可以的话，您可以尝试写一款工具，如果有需要，请留下您的邮箱，我们可以协助你。
工具开发参考文档：https://docs.he3app.com/guide/

**Comment by Asura88** [comment]
我自己不会写，麻烦你们帮我写一个模版，比如注入漏洞修复代码是`123`，你们给我提供一个基本模版，我只需要照着添加修复代码即可
